### PR TITLE
add fetchCurrentAuthenticatedUser action to cognito module

### PR DESCRIPTION
**Pull request by jonalexander**
- allows us to check for current auth'd user cached in browser

### CODE REVIEW

File: packages/cognito-module/src/actions.js [hunk 1]
```diff
@@ -2,6 +2,16 @@ import Auth from '@aws-amplify/auth'
 import Amplify from '@aws-amplify/core'
 
 export default {
+  fetchCurrentAuthenticatedCuser: ({ commit }) => {
```
Comment (bdeo) [inline packages/cognito-module/src/actions.js]: There's a typo here. fetchCurrentAuthenticated-C-user

Comment (jonalexander) [reply packages/cognito-module/src/actions.js]: oof sorry. fixed.

**Comment by jonalexander** [comment]
@bdeo can you review again? thank you.

**Comment by jonalexander** [comment]
@bdeo

**Comment by jonalexander** [comment]
@bdeo

**Pull request merged**